README.md

```diff
@@ -423,6 +423,24 @@ end
 <%= component "button", label: "Sign in", url: sign_in_path %>
 ```
 
+### Validation
+
+To ensure your components get initialized properly you can use ActiveModel::Validations in your elements or components:
```

Comment:
Perhaps backticks around `ActiveModel::Validations` (and `ActiveModel::ValidationError` further down)

Reply:
Sure - will do.